pkg/osquery/tables/gsettings/gsettings.go

```diff
@@ -0,0 +1,184 @@
+// +build !windows
+
+package gsettings
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	osquery "github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+const gsettingsPath = "/usr/bin/gsettings"
+
+const allowedCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-."
+
+type gsettingsExecer func(ctx context.Context, username string, buf *bytes.Buffer) error
+
+type GsettingsValues struct {
+	client   *osquery.ExtensionManagerClient
+	logger   log.Logger
+	getBytes gsettingsExecer
+}
+
+// Settings returns a table plugin for querying setting values from the
+// gsettings command.
+func Settings(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	columns := []table.ColumnDefinition{
+		table.TextColumn("schema"),
+		table.TextColumn("key"),
+		table.TextColumn("value"),
+		table.TextColumn("user"),
```

Comment:
Other tables, like touchid, we use `uid` for the numeric uid. I don't have a strong feeling about it, but I think we should aim to be consistent. @FritzX6 do you have an opinion?

Reply:
here, "user" is actually the user name, not the uid. Renamed the column to be more clear, but if folks feel strongly we can change the column to be a uid.